### PR TITLE
Add link to Registration app on no systems page

### PR DESCRIPTION
**Pull request by mhuth**
## Prerequisites

- [?] Scope your styles to your individual card
- [x] You are using translations when necessary with proper plural/singular modifications
- [x] You are using the template card provided
- [NA] Use functional components & hooks (please)
- [x] Attach screenshots (before and after)
- [?] Make sure that all text in blue is populated with the correct link. If you're unsure what the url should be, just ask!

## What
Adds a button/link to the new Registration app (this PR points to https://cloud.redhat.com/beta/insights/registration) on the NoSystems page.  Removes the commands on that page for installing/registering systems coz that is now in the Registration app.  Removes the button to the documentation coz that too is in the Registration app.

### Before
no_systems_page_before

### After
no_systems_page_after

## Additional Info
https://projects.engineering.redhat.com/browse/RHCLOUD-8019

### CODE REVIEW

**Comment by mhuth** [comment]
Reviews please @AllenBW @kybaker !  Don't seem to have sufficient perms to officially add you as reviewers to the PR.

File: src/PresentationalComponents/NoSystems/NoSystems.js [hunk 1]
```diff
@@ -34,25 +33,13 @@ const NoSystems = () => {
...
-                            <ClipboardCopy>insights-client --register</ClipboardCopy>
-                        </StackItem>
                     </Stack>
                 </EmptyStateBody>
                 <Button
                     component="a"
-                    href="https://access.redhat.com/products/red-hat-insights#getstarted"
-                    target="_blank" variant="primary">
-                    {intl.formatMessage(messages.noSystemsLink)}
+                    href="https://cloud.redhat.com/beta/insights/registration"
```
Comment (AllenBW) [inline src/PresentationalComponents/NoSystems/NoSystems.js:40]: OOOOO good point this is only on beta atm 🤔 guess we'll do another pr to update this url once it shows up on stable
